examples/datadogagent/datadog-agent-openshift.yaml

```diff
@@ -2,26 +2,33 @@ apiVersion: datadoghq.com/v1alpha1
 kind: DatadogAgent
 metadata:
   name: datadog
+  namespace: datadog
 spec:
   credentials:
     apiKey: <paste-your-api-key-here>
     appKey: <paste-your-app-key-here>
   agent:
+    rbac:
+      serviceAccountName: datadog-agent-scc
     config:
+      securityContext:
+        runAsUser: 0
+        seLinuxOptions:
+          level: s0
+          role: system_r
+          type: spc_t
+          user: system_u
       criSocket:
         criSocketPath: /var/run/crio/crio.sock
-        useCriSocketVolume: true
       env:
       - name: DD_KUBELET_TLS_VERIFY
         value: "false"
     log:
-      enabled: false
+      enabled: true
+    apm:
+      enabled: true
+    process:
+      enabled: true
+      processCollectionEnabled: true
   clusterAgent:
-    config:
-      externalMetrics:
-        enabled: true
-        port: 5010
-      clusterChecksRunnerEnabled: true
-    replicas: 2
-  clusterChecksRunner:
-    enabled: true
+    replicas: 1
```

Comment:
this part is not mandatory. 
the clusterAgent is enable by default with 1 replica

Reply:
removed